### ISSUE TITLE
Output_bonds script doesn't give information on timestep [BUG] XXX

### ISSUE THREAD

**Issue by Prateek1410** [lorenzo-rovigatti/oxDNA#47]
Hello!

For every configuration, the output_bonds shows t =0. Everything else seems to be the alright, based on the size, I think it has generated data completely just the time info is missing. Please help me with this, I need to run this command for a big set of data. 

**To Reproduce**
I ran this command: `oat output_bonds input.txt trajectory.dat | tee output_bonds.txt >/dev/null 2>&1 &`

**Desktop (please complete the following information):**
 - I use WSL (Ubuntu 20.04) on windows 10.
 - oxDNA_analysis_tools version: 2.0.3
- cmake version 3.16.3

Warm Regards
Prateek

**Comment by ErikPoppleton** [comment]
@lorenzo-rovigatti I believe this is a problem with the `pair_energy` observable, because that line that shows `t=0` is being printed straight from the observable.  I vaguely remember we discussed this many months ago but can't remember what the resolution was.

**Comment by ErikPoppleton** [comment]
I see, thank you!

**Comment by lorenzo-rovigatti** [comment]
You are very right Erik. `output_bonds.py` passed `0` as the current step to the observable. I have fixed it by passing `backend.config_info().current_step`.

Note that the code has changed a lot since we first implemented observables, which could now get access to the current step by using `ConfigInfo`'s `curr_step` member. This is a change that I should be doing to the C++ code in the future as soon as I have some time.
